pre_commit_dbt/check_macro_is_referenced.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Sequence
+
+from pre_commit_dbt.utils import add_filenames_args
+from pre_commit_dbt.utils import add_manifest_args
+from pre_commit_dbt.utils import get_json
+from pre_commit_dbt.utils import get_macro_sqls
+from pre_commit_dbt.utils import get_macros
+from pre_commit_dbt.utils import JsonOpenError
+
+def get_macro_name(macros_ref):
+    return map(lambda x:x.split(".")[-1], macros_ref) 
```

Comment:
Maybe add some type hinting if it makes sense to you here ? Also, do you think it could be re-used for other hooks or is it really specific to yours? Seems to me like it could be re-users and then maybe placed in utils but you tell me.